docs/content/references/sui-api/graphql/guides.mdx

```diff
@@ -0,0 +1,257 @@
+
+
+#### Example 1: Get total transaction blocks
+The goal is to get the total number of transaction blocks in the network.
+
+<Tabs groupId="migrating-json-rpc">
+
+<TabItem value="json-rpc" label="JSON-RPC">
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "sui_getTotalTransactionBlocks",
+  "params": []
+}
+```
+
+</TabItem>
+<TabItem value="graphql" label="GraphQL">
+
+```graphql
+{
+  checkpoint {
+    networkTotalTransactions
+  }
+}
+```
+
+</TabItem>
+</Tabs>
+
+#### Example 2: Get a specific transaction block
+The goal is to get the transaction block by its digest
+
+<Tabs groupId="migrating-json-rpc">
+
+<TabItem value="json-rpc" label="JSON-RPC">
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "sui_getTransactionBlock",
+  "params": [
+    "Hay2tj3GcDYcE3AMHrej5WDsHGPVAYsegcubixLUvXUF",
+    {
+      "showInput": true,
+      "showRawInput": false,
+      "showEffects": true,
+      "showEvents": true,
+      "showObjectChanges": false,
+      "showBalanceChanges": false
+    }
+  ]
+}
+```
+
+</TabItem>
+<TabItem value="graphql" label="GraphQL">
+
+```graphql
+{
+  transactionBlock(digest: "Hay2tj3GcDYcE3AMHrej5WDsHGPVAYsegcubixLUvXUF") {
+    gasInput {
+      gasSponsor {
+        address
+      }
+      gasPrice
+      gasBudget
+    }
+    effects {
+      status
+      timestamp
+      checkpoint {
+        sequenceNumber
+      }
+      epoch {
+        epochId
+        referenceGasPrice
+      }
+    }
+  }
+}
+```
+</TabItem>
+</Tabs>
+
+#### Example 3: Get coin objects owned by an address
+The goal is to return all `Coin<coin_type>` objects owned by an address.
+
+<Tabs groupId="migrating-json-rpc">
+
+<TabItem value="json-rpc" label="JSON-RPC">
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "suix_getCoins",
+  "params": [
+    "0x5094652429957619e6efa79a404a6714d1126e63f551f4b6c7fb76440f8118c9", //owner
+    "0x2::sui::SUI",                                                      //coin type
+    "0xe5c651321915b06c81838c2e370109b554a448a78d3a56220f798398dde66eab", //cursor
+    3 //limit
+  ]
+}
+```
+
+</TabItem>
+<TabItem value="graphql" label="GraphQL">
+
+```graphql
+{
+  address(address: "0x5094652429957619e6efa79a404a6714d1126e63f551f4b6c7fb76440f8118c9") {
+    coinConnection(
+      first: 3, 
+      after: "0x5094652429957619e6efa79a404a6714d1126e63f551f4b6c7fb76440f8118c9", 
+      type: "0x2::sui::SUI"
+    ) {
+      nodes {
+        asMoveObject {
+          asCoin {
+            address
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+:::info
+Observe that the cursor is now passed in the `after` (or `before`) fields on the connection, and the limit in the `first` or `last` fields.
+:::
+
+</TabItem>
+</Tabs>
+
+### Support in TS SDK
+TBD
```

Comment:
As commented, the page will be reachable by search. If you're not okay with this section displaying as is, remove and replace when content is available.